tests/test_live.py

```diff
@@ -87,7 +87,9 @@ def test_instruments():
         host=get_platform_dispatcher(),
         instrument="mock",
     )
-    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'gw', 'spi_acs', 'legacysurvey']
+    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'gw', 'spi_acs', 'legacysurvey',
+                                           'lightcurve-example', 'cta-example', 'gaia-andrii', 'fermi', 'mmoda-fink-backend'
```

Comment:
this is going to break every time workflow developers add new instruments. maybe it's better to do 
```suggestion
 assert set(['isgri', 'jemx', 'polar', 'antares', 'gw', 'spi_acs', 'legacysurvey']) - set(disp.get_instruments_list()) == set()```